docs/resources/windows_firewall.md.erb

```diff
@@ -0,0 +1,90 @@
+---
+title: About the windows_firewall Resource
+platform: windows
+---
+
+# windows_firewall
+
+Use the `windows_firewall` Chef InSpec audit resource to test if a firewall profile is correctly configured on a Windows system.
+
+<br>
+
+## Availability
+
+### Installation
+
+This resource is distributed along with Chef InSpec itself. You can use it automatically.
+
+### Version
```

Comment:
I believe this section can be omitted for the resource's initial commit; so this PR does not need to chase after a moving InSpec version number.

We should change the contributor docs here, because it's confusing to ask people to provide an unpredictable version number.